migrations/9_espumita_ucok_tarkus_hook.js

```diff
@@ -9,36 +9,35 @@ const tokenIdUcok = "14";
 const tokenIdTarkus = "15";
```

Comment:
Not relevant, just house keeping stuff, to organise our previous animal launches.